plugins/modules/remote_management/redfish/ilo_redfish_info.py

```diff
@@ -178,9 +257,44 @@ def main():
             for command in command_list:
                 if command == "GetiLOSessions":
                     result[command] = rf_utils.get_ilo_sessions()
+        elif category == "Systems":
+            resource = rf_utils._find_systems_resource()
+            if resource['ret'] is False:
+                result[command] = resource
+                module.fail_json(msg=to_native(result))
+
+            for command in command_list:
+                if command == "GetServiceBiosAttributes":
+                    result[command] = rf_utils.get_service_bios_attributes()
+                elif command == "GetBootSettings":
+                    result[command] = rf_utils.get_network_boot_settings()
+                elif command == "GetPhysicalDrives":
+                    result[command] = rf_utils.get_physical_drives_details()
+                elif command == "GetLogicalDrives":
+                    result[command] = rf_utils.get_logical_drives_details()
+                elif command == "GetLogicalDrivesWithArrayControllers":
+                    result[command] = rf_utils.get_logical_drives_details(True)
+                elif command == "GetServerPostState":
+                    result[command] = rf_utils.get_server_poststate()
+                elif command == "GetNvmeDrives":
+                    result[command] = rf_utils.get_nvme_drives_details()
+        elif category == "Manager":
+            resource = rf_utils._find_managers_resource()
+            if not resource["ret"]:
+                result[command] = resource
+                module.fail_json(msg=to_native(result))
+
+            for command in command_list:
+                if command == "GetSNMPv3Users":
+                    result[command] = rf_utils.get_snmpv3_users()
+                elif command == "GetSNMPAlertDestinations":
+                    result[command] = rf_utils.get_snmp_alert_destinations()
+
+    if not result[command]['ret']:
+        module.fail_json(msg=to_native(result))
 
     module.exit_json(ilo_redfish_info=result)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
Please keep in mind that new modules are required to provide tests, as described in:
https://github.com/ansible-collections/community.general/blob/main/CONTRIBUTING.md#creating-new-modules-or-plugins